io.github.quodlibet.ExFalso.yaml

```diff
@@ -0,0 +1,100 @@
+---
+app-id: io.github.quodlibet.ExFalso
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+command: exfalso
+build-options:
+  no-debuginfo: true
+  strip: true
+finish-args:
+  - "--share=network"
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--filesystem=host"
+
+  # some embedded covers and the fifo is still shared through /tmp
+  - "--filesystem=/tmp"
+
+  # bookmarks in the filesystem browser
+  - "--filesystem=xdg-config/gtk-3.0:ro"
+
+  # ...detecting the running desktop environment
+  - "--talk-name=com.canonical.Unity.Launcher"
+  - "--talk-name=org.enlightenment.wm.service"
+  - "--talk-name=org.kde.plasmashell"
+
+  # ...various other plugins
+  - "--talk-name=org.freedesktop.FileManager1"
+  - "--talk-name=org.gnome.SessionManager"
+  - "--talk-name=org.xfce.FileManager"
+
+  # ...talking to dconf
```

Comment:
Dconf permissions are not needed anymore

Reply:
ok, thanks, I'll have a look

Reply:
works nicely, thanks